.github/CONTRIBUTING.md

```diff
@@ -90,8 +90,9 @@ A sentence describing the type of change with necessary and relevant information
 
 #### Change type
 * **Feature**: Added a new feature.
+* **Changed:**: Changes to already existing features.
 * **Fixed**: When there is a hotfix or bugfix. Something has been upgraded could be mentioned too.
```

Comment:
Can we append `Something has been upgraded could be mentioned too.` text with "Changed:" type mentioned above?  I feel that we also could avoid using `**Changed:**:` keyword. I am still finding a good keyword.

Reply:
I don't think that `Fixed` is the right usage for this. `Fixed` means that there was a bug or some behavior that was incorrectly. There are some cases where the behavior was correct as originally defined, but something was changed due to product or technical requirements. So I would prefer to also remove the ` Something has been upgraded could be mentioned too.` line from `Fixed:`.

I don't think `Chore` and `Refactor` should be added because these types of tasks are only relevant to the SDK developers, not to the users of the SDK. If we did the refactoring correctly, then the SDK user shouldn't notice that there's been any refactoring.

Maybe we can add an `Upgrade:` for things like dependency and version upgrades? And then change Feature to this: `**Feature:** Added a new feature or changed an existing feature`.

What do you think?

Reply:
> So I would prefer to also remove the "Something has been upgraded could be mentioned too." line from Fixed:

I agree. 

> I don't think Chore and Refactor should be added because these types of tasks are only relevant to the SDK developers, not to the users of the SDK

Makes sense!

> Maybe we can add an Upgrade: for things like dependency and version upgrades? And then change Feature to this: **Feature:** Added a new feature or changed an existing feature

Yep (+1).